src/components/MapEmbed.js

```diff
@@ -23,7 +32,8 @@ import { select } from '@wordpress/data';
 const MapEmbed = (props) => {
 	const isAdmin = select('core')?.canUser('create', 'posts');
 	const isPostEditor = Boolean(select('core/edit-post'));
-	const { zoom, type, className } = props;
+	const mapPlatform = getFromGlobal('settings.mapPlatform');
+	const { zoom, type, className, latitude, longitude } = props;
```

Comment:
mapPlatform should have a default as a fallback for people like me, wondering about a missing map, because they & I never saved 'GatherPress >> Settings' (after I installed that new version aka PR)

Reply:
Fixed! Thank you for catching this